geospaas_harvesting/ingesters.py

```diff
@@ -258,7 +276,8 @@ class MetanormIngester(Ingester):
         'location_geometry',
         'provider',
         'iso_topic_category',
-        'gcmd_location'
+        'gcmd_location',
+        'geospaas_parameters'
```

Comment:
We are already in the context of geospaas, so it's not really useful to specify it again here
```suggestion
        'parameters'
```